Applications/Calculator/CalculatorWidget.cpp

```diff
@@ -189,17 +155,15 @@ void CalculatorWidget::add_operation_button(GUI::Button& button, Calculator::Ope
 
 void CalculatorWidget::add_digit_button(GUI::Button& button, int digit)
 {
-    add_button(button, Color::Blue);
-    button.set_text(String::number(digit));
+    set_btn_color(button, Color::Blue);
     button.on_click = [this, digit](auto) {
         m_keypad.type_digit(digit);
         update_display();
     };
 }
 
-void CalculatorWidget::add_button(GUI::Button& button, Color text_color)
+void CalculatorWidget::set_btn_color(GUI::Button& button, Color text_color)
 {
-    button.resize(35, 28);
     auto palette = button.palette();
     palette.set_color(Gfx::ColorRole::ButtonText, text_color);
     button.set_palette(palette);
```

Comment:
This can be done in GML as well now that we have `foreground_color`.
Also we don't use random abbreviations like `btn` in this codebase. Instead, use `button`.